keras_core/ops/nn.py

```diff
@@ -22,6 +22,24 @@ def compute_output_spec(self, x):
 
 @keras_core_export(["keras_core.ops.relu", "keras_core.ops.nn.relu"])
 def relu(x):
+    """Rectified linear unit activation function.
+
+    It is defined as `f(x) = max(0, x)`.
+
+    Args:
+        x: Input tensor.
+
+    Returns:
+        A tensor with the same shape as `x`.
+
+    Examples:
+
+        >>> x = np.array([-1, 0, 1])
```

Comment:
1. Unindent (the `>>>` should be at docstring level
2. Prefer using floats since that's the typical activation use case